src/main/resources/TotemCode/TotemMacros.xml

```diff
@@ -311,5 +311,27 @@
 &lt;input type="hidden" id="redirectURL" value="$doc.getURL('view','showEdit=true')"/&gt;
 &lt;input type="hidden" id="showIframe" value="1"/&gt;
 #end
+
+#**
+* Display the totem actions links : Edit totem, documentation ...
+*#
+#macro(displayTotemActionsLinks)
+#set($totemViewURL = $doc.getURL())
+#set($totemEditURL = $doc.getURL('view','showEdit=true'))
+#set($totemDocURL = $xwiki.getURL('TotemCode.Documentation','view'))
+{{html clean="false" wiki="true"}}
+&lt;div class="clearfix"&gt;
+&lt;div class="pull-right"&gt;
+ &lt;a class="btn btn-info" href="$totemDocURL" target="_blank"&gt;&lt;span class="glyphicon glyphicon-book"&gt;&lt;/span&gt; $services.localization.render('totem.edit.documentation')&lt;/a&gt;
```

Comment:
The problem is icons are not necessary images. They could be characters in an icon font (like glyphicon or font-awesome).

What you need to do is replacing:
```
&lt;a class="btn btn-info" href="$totemDocURL" target="_blank"&gt;&lt;span class="glyphicon glyphicon-book"&gt;&lt;/span&gt; $services.localization.render('totem.edit.documentation')&lt;/a&gt;
```
by:
```
&lt;a class="btn btn-info" href="$totemDocURL" target="_blank"&gt;$services.icon.renderHTML('book') $services.localization.render('totem.edit.documentation')&lt;/a&gt;
```

More infos on http://extensions.xwiki.org/xwiki/bin/view/Extension/Icon+Theme+Application.